config/initializers/devise.rb

```diff
@@ -250,7 +250,16 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-
+  config.omniauth :shibboleth,
+                  # TODO: mappings will need to change when authenticating against IST's SAML
+                  uid_field: 'eppn',
```

Comment:
We should just consolidate those two configs into one? Pretty much the second one:
```ruby
  config.omniauth :shibboleth, {
    :uid_field => ->(request) {request.call('uid') || request.call('eppn').gsub(/@.*/, '')},
    :info_fields => {:name => 'givenName', :last_name => 'sn'}
  } 
```
Then it will work in any environment. If uid not available (testshib) will fall back to eppn.